src/EPR.Calculator.Frontend/appsettings.Development.json

```diff
@@ -17,6 +17,10 @@
     "RunParameterYear": "2024-25",
     "DashboardCalculatorRunApi": "http://localhost:5055/v1/calculatorRuns"
   },
+  "CalculatorRun": {
+    "RunParameterYear": "2024-25",
```

Comment:
Trilok /  @johnbenjamin-defra  Can use existing RunParameterYear? Do we need another one for Run? Please confirm.

Reply:
@udaydefra I looked at the code and we are using `parameter year` all throughout the configs.

We can leave it for now and definitely need a ticket Post December 2024 when the year selection dropdown comes into action :)

Reply:
Thanks @johnbenjamin-defra